addon/resources/country-codes.js

```diff
@@ -1,224 +1,2057 @@
+    alpha3: 'PRY',
+    countryCallingCodes: ['595'],
+    currencies: ['PYG'],
+    name: 'Paraguay'
+  },
+  {
+    id: 'QA',
+    alpha2: 'QA',
+    alpha3: 'QAT',
+    countryCallingCodes: ['974'],
+    currencies: ['QAR'],
+    name: 'Qatar'
+  },
+  {
+    id: 'RE',
+    alpha2: 'RE',
+    alpha3: 'REU',
+    countryCallingCodes: ['262'],
+    currencies: ['EUR'],
+    name: 'Reunion'
+  },
+  {
+    id: 'RO',
+    alpha2: 'RO',
+    alpha3: 'ROU',
+    countryCallingCodes: ['40'],
+    currencies: ['RON'],
+    name: 'Romania'
+  },
+  {
+    id: 'RS',
+    alpha2: 'RS',
+    alpha3: 'SRB',
+    countryCallingCodes: ['381'],
+    currencies: ['RSD'],
+    name: 'Serbia'
+  },
+  {
+    id: 'RU',
+    alpha2: 'RU',
+    alpha3: 'RUS',
+    countryCallingCodes: ['7', '7 3', '7 4', '7 8'],
+    currencies: ['RUB'],
+    name: 'Russian Federation'
+  },
+  {
+    id: 'RW',
+    alpha2: 'RW',
+    alpha3: 'RWA',
+    countryCallingCodes: ['250'],
+    currencies: ['RWF'],
+    name: 'Rwanda'
+  },
+  {
+    id: 'SA',
+    alpha2: 'SA',
+    alpha3: 'SAU',
+    countryCallingCodes: ['966'],
+    currencies: ['SAR'],
+    name: 'Saudi Arabia'
+  },
+  {
+    id: 'SB',
+    alpha2: 'SB',
+    alpha3: 'SLB',
+    countryCallingCodes: ['677'],
+    currencies: ['SBD'],
+    name: 'Solomon Islands'
+  },
+  {
+    id: 'SC',
+    alpha2: 'SC',
+    alpha3: 'SYC',
+    countryCallingCodes: ['248'],
+    currencies: ['SCR'],
+    name: 'Seychelles'
+  },
+  {
+    id: 'SD',
+    alpha2: 'SD',
+    alpha3: 'SDN',
+    countryCallingCodes: ['249'],
+    currencies: ['SDG'],
+    name: 'Sudan'
+  },
+  {
+    id: 'SE',
+    alpha2: 'SE',
+    alpha3: 'SWE',
+    countryCallingCodes: ['46'],
+    currencies: ['SEK'],
+    name: 'Sweden'
+  },
+  {
+    id: 'SG',
+    alpha2: 'SG',
+    alpha3: 'SGP',
+    countryCallingCodes: ['65'],
+    currencies: ['SGD'],
+    name: 'Singapore'
+  },
+  {
+    id: 'SH',
+    alpha2: 'SH',
+    alpha3: 'SHN',
+    countryCallingCodes: ['290'],
+    currencies: ['SHP'],
+    name: 'Saint Helena, Ascension And Tristan Da Cunha'
+  },
+  {
+    id: 'SI',
+    alpha2: 'SI',
+    alpha3: 'SVN',
+    countryCallingCodes: ['386'],
+    currencies: ['EUR'],
+    name: 'Slovenia'
+  },
+  {
+    id: 'SJ',
+    alpha2: 'SJ',
+    alpha3: 'SJM',
+    countryCallingCodes: ['47'],
+    currencies: ['NOK'],
+    name: 'Svalbard And Jan Mayen'
+  },
+  {
+    id: 'SK',
+    alpha2: 'SK',
+    alpha3: 'SVK',
+    countryCallingCodes: ['421'],
+    currencies: ['EUR'],
+    name: 'Slovakia'
+  },
+  {
+    id: 'SL',
+    alpha2: 'SL',
+    alpha3: 'SLE',
+    countryCallingCodes: ['232'],
+    currencies: ['SLL'],
+    name: 'Sierra Leone'
+  },
+  {
+    id: 'SM',
+    alpha2: 'SM',
+    alpha3: 'SMR',
+    countryCallingCodes: ['378'],
+    currencies: ['EUR'],
+    name: 'San Marino'
+  },
+  {
+    id: 'SN',
+    alpha2: 'SN',
+    alpha3: 'SEN',
+    countryCallingCodes: ['221'],
+    currencies: ['XOF'],
+    name: 'Senegal'
```

Comment:
can you add a `bestCountry: true` here ?

Reply:
Indeed, we can add a `showOnTop: true | false` to add them to the top of the list when displaying.